mycpp/demo/gc_heap.cc

```diff
@@ -276,6 +276,8 @@ class AtomEntry {
 class AtomTable {
```

Comment:
Hm I think this whole thing is just unused.  

Ditto for the Local<T> and Param<T> thing.  We could either delete those or move them to an experimental file if it's confusing.

Reply:
I'm unbothered .. I got link errors when I checked if these things are called and didn't feel like digging deeper.  Might be worth getting rid of them if we're not planning on using them for anything.